docs/modules/ROOT/pages/documentation/payara-server/notification-service/notifiers.adoc

```diff
@@ -5,22 +5,11 @@ The notification service comes with an integrated set of notifiers that
 can be used to send notification events to a distinct variety of
 channels apart from the standard server log mechanism:
 
-*Email*:: Taking advantage of preconfigured Java Mail sessions in the
-server, you can route notifications to an email account
 *JMS*:: Taking advantage of preconfigured JMS connection resources, you
 can route notifications to a Java Message Queue so it can be consumed by
 other applications or services
 *CDI Event Bus*:: For sending notification events using xref:/documentation/payara-server/public-api/cdi-events.adoc[Payara's CDI Event Bus]
  and observing them in instances in the cluster.
-*Hipchat*:: You can configure the notification service to send events
-to an specific Hipchat chat room.
-*Slack*:: Same as with the Hipchat notifier, you can sends
-notifications directly to an specific Slack channel.
-*SNMP*:: Allows monitoring servers to scout for notification events
-using SNMP traps.
-*XMPP*:: Allows sending notification events to XMPP servers.
-*NewRelic*:: Allows sending notification events to a *NewRelic* service account.
-*Datadog*:: Allows sending notification events to the *Datadog* monitoring service
```

Comment:
Should we insert a link to the enterprise documentation indicating it has more notifiers available out of the box?

Reply:
Sure, I didn't think about it. I'll do it.